desktop-app/src/main/main.ts

```diff
@@ -119,7 +125,9 @@ const createWindow = async () => {
     width,
     height,
     icon: getAssetPath('icon.png'),
+    titleBarStyle: 'hidden',
     webPreferences: {
+      sandbox: false,
```

Comment:
Why do we want to disable the `sandbox` option?

Reply:
The `custom-electron-titlebar` mentioned it in their [documentation](https://github.com/AlexTorresDev/custom-electron-titlebar/wiki/Get-Started), for the library to function properly, `sandbox` should be `false`.

However, I tested the functionality by **omitting** the `sandbox` line, and found out, the custom titlebar works just fine and encountered no issues yet. 

Should I keep this line anyways?

Reply:
If it works without this, then I would suggest we remove this, as this is option is a compromise on security.